cocos2dx/CMakeLists.txt

```diff
@@ -187,7 +189,7 @@ if(OHOS)
 	    -Wno-unsequenced
 	    -Wno-extra
 	    -Wno-c++11-narrowing
-	    -DOpenHarmony
+	    -DHarmonyOS_Next
```

Comment:
Letters in macros should be all uppercase to keep the same coding style, for example `USE_FILE32API`, `CC_TARGET_PLATFORM`.